colabs/Wczytanie_danych.ipynb

```diff
@@ -174,6 +174,96 @@
         }
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#Załadowanie repozytorium"
+      ],
+      "metadata": {
+        "id": "lO5bbY2iww-u"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##Załadowanie repozytorium"
+      ],
+      "metadata": {
+        "id": "inq92uX-w20K"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!git clone https://github.com/PiotrMaciejKowalski/BigData2022-actors.git"
+      ],
+      "metadata": {
+        "id": "5EkoqzLSw4Li",
+        "outputId": "a84986c1-af10-4694-c755-79cc27a71bb3",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        }
+      },
+      "execution_count": 4,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Cloning into 'BigData2022-actors'...\n",
+            "remote: Enumerating objects: 294, done.\u001b[K\n",
+            "remote: Counting objects: 100% (221/221), done.\u001b[K\n",
+            "remote: Compressing objects: 100% (168/168), done.\u001b[K\n",
+            "remote: Total 294 (delta 81), reused 146 (delta 53), pack-reused 73\u001b[K\n",
+            "Receiving objects: 100% (294/294), 4.44 MiB | 20.75 MiB/s, done.\n",
+            "Resolving deltas: 100% (95/95), done.\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##Testujemy dostępność do metod z katalogu tests"
+      ],
+      "metadata": {
+        "id": "-zuhXzW3mDFT"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import sys"
+      ],
+      "metadata": {
+        "id": "1WzWVfbrzF3X"
+      },
+      "execution_count": 5,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "sys.path.append('/content/BigData2022-actors/tests')\n",
```

Comment:
sys path append wygląda okropnie. Jeśli klonowanie zadziałało dobrze powinno wystarczyć
```
from test.simple_test import *
```
lub
```
from .test.simple_test import *
```

Jeśli nie to - po git clone spróbowałbym jeszcze komendy przeniesienia plików
```
mv Katalog_z_kodem/* .  
```

Reply:
Poprawione

Reply:
nie, nie do końca. 
```
!mv /content/BigData2022-actors/tests/* .
```
Przenosi tylko katalag tests, a nie całość projektu.

Powinno być 
```
!mv /content/BigData2022-actors/* .
```

Reply:
Spróbowałem tych kawałków kodu które Pan mi pisał w komentarzach i nie byłem w stanie otrzymać zamierzonego celu.
Nie jestem w stanie znaleźć innej metody na importy tych plików, niż zaimportowanie pojedynczego pliku "simple_test" z folderu "tests", a następnie uruchomienie metody z tego pliku poprzez plik.metoda. Tak właśnie teraz wygląda import.
Przy próbie załadowania wszystkich plików poprzez * nie jestem w stanie uruchomić metody z pliku "simple_test", gdyż go nie rozpoznaje.

Reply:
Poprawka. Dwie pierwsze linijki które Pan zaproponował nawet się nie kompilują. Próbowałem zmieniać "." na "/" i bawić się dalej z lokalizacją z której importuję, jednak to mi nie wychodzi.

Reply:
Jeszcze raz do tego usiadłem, bo powinno to działać i teraz już działa tak jak powinno.